faq.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8" />
-  <title>AntennaPod | Knowledge Base</title>
+  <title>AntennaPod | F.A.Q.</title>
```

Comment:
Yeah, you mentioned that before. Think that's a great idea, but I guess that should be done on a separate page (otherwise this would be too cluttered/long). Then together they then make up the knowledge base; dunno if you had a vision yet on how the different parts would link together?